pkgs/servers/web-apps/matomo/default.nix

```diff
@@ -3,16 +3,16 @@
 let
   versions = {
     matomo = {
-      version = "3.14.1";
-      sha256 = "0gp6v797118z703nh0p77zvsizvdg0c2jkn26996d4sqw5pa78v3";
+      version = "4.2.1";
+      sha256 = "d3ea7572c5b42f2636da89b9c15dd7ae16da1d06dab0cea2ed93304a960277ac";
     };
 
     matomo-beta = {
```

Comment:
This should be moved to `aliases.nix` if it doesn not actually refer to a beta anymore.

Reply:
there are still betas ex. 4.3.0-b2 was released a few days ago but I haven't even finished the PR I started in december :(